src/hound/config/config.go

```diff
@@ -19,10 +19,11 @@ type UrlPattern struct {
 }
 
 type Repo struct {
-	Url            string      `json:"url"`
-	MsBetweenPolls int         `json:"ms-between-poll"`
-	Vcs            string      `json:"vcs"`
-	UrlPattern     *UrlPattern `json:"url-pattern"`
+	Url            string          `json:"url"`
+	MsBetweenPolls int             `json:"ms-between-poll"`
+	Vcs            string          `json:"vcs"`
+	VcsConfig      json.RawMessage `json:"vcs-config"`
```

Comment:
Yes, but I want to land the svn driver first because it's going to be the first one to use the vcs-config.